diktat-rules/src/main/kotlin/org/cqfn/diktat/ruleset/utils/AstNodeUtils.kt

```diff
@@ -723,6 +723,19 @@ fun ASTNode.extractLineOfText(): String {
     return text.joinToString(separator = "").trim()
 }
 
+/**
+ * @param pred a predicate
+ * @return filtered sequence
+ */
+private fun <T> Sequence<T>.takeWhileInclusive(pred: (T) -> Boolean): Sequence<T> {
```

Comment:
have to move to this place because of a bug in `kotlin-maven-plugin` (will look at it later)